src/qps.jl

```diff
@@ -1,4 +1,13 @@
-function QuadraticModel(qps::QPSData)
-    QuadraticModel(qps.c, qps.Q, opHermitian(qps.Q), qps.A,
-                   qps.lcon, qps.ucon, qps.lvar, qps.uvar, c0=qps.c0)
+using QPSReader
```

Comment:
If I remove the line `using QPSReader` I get this error message when testing:
 
`ERROR: LoadError: LoadError: MethodError: no method matching QuadraticModel(::QPSData, ::Array{Float64,1})
Closest candidates are:
  QuadraticModel(::Any, ::Any, ::Any) at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\src\qpmodel.jl:17
  QuadraticModel(::AbstractNLPModel, ::AbstractArray{T,1} where T; kwargs...) at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\src\qpmodel.jl:83
Stacktrace:
 [1] uncqp_QPSData() at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\problems\uncqp.jl:35
 [2] top-level scope at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\test_consistency.jl:37
 [3] include(::String) at .\client.jl:439
 [4] top-level scope at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\runtests.jl:18
 [5] include(::String) at .\client.jl:439
 [6] top-level scope at none:6
in expression starting at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\test_consistency.jl:33
in expression starting at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\runtests.jl:18
ERROR: Package QuadraticModels errored during testing`

Reply:
It's because of `@require`. Try importing `QPSReader` before `QuadraticModels`:
```
using QPSReader
using QuadraticModel  # qps.jl is included
```

Reply:
You mean in the runtest.jl file?
If I do: 
```
using QPSReader
using LinearAlgebra, LinearOperators, NLPModels, QuadraticModels
```

I get this error:
```
┌ Warning: Error requiring QPSReader from QuadraticModels:
│ LoadError: UndefVarError: QPSData not defined
│ Stacktrace:
│  [1] top-level scope at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\src\qps.jl:2
│  [2] include(::Module, ::String) at .\Base.jl:377
│  [3] include(::String) at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\src\QuadraticModels.jl:1
│  [4] top-level scope at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\runtests.jl:6
│  [5] eval at .\boot.jl:331 [inlined]
│  [6] eval at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\src\QuadraticModels.jl:1 [inlined]
│  [7] (::QuadraticModels.var"#16#19")() at C:\Users\Geoffroy Leconte\.julia\packages\Requires\9Jse8\src\require.jl:67
│  [8] err(::QuadraticModels.var"#16#19", ::Module, ::String) at C:\Users\Geoffroy Leconte\.julia\packages\Requires\9Jse8\src\require.jl:38
│  [9] #15 at C:\Users\Geoffroy Leconte\.julia\packages\Requires\9Jse8\src\require.jl:66 [inlined]
│  [10] withpath(::QuadraticModels.var"#15#18", ::String) at C:\Users\Geoffroy Leconte\.julia\packages\Requires\9Jse8\src\require.jl:28
│  [11] #14 at C:\Users\Geoffroy Leconte\.julia\packages\Requires\9Jse8\src\require.jl:65 [inlined]
│  [12] listenpkg(::QuadraticModels.var"#14#17", ::Base.PkgId) at C:\Users\Geoffroy Leconte\.julia\packages\Requires\9Jse8\src\require.jl:13
│  [13] macro expansion at C:\Users\Geoffroy Leconte\.julia\packages\Requires\9Jse8\src\require.jl:64 [inlined]
│  [14] __init__() at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\src\QuadraticModels.jl:23
│  [15] _include_from_serialized(::String, ::Array{Any,1}) at .\loading.jl:697
│  [16] _require_search_from_serialized(::Base.PkgId, ::String) at .\loading.jl:781
│  [17] _require(::Base.PkgId) at .\loading.jl:1006
│  [18] require(::Base.PkgId) at .\loading.jl:927
│  [19] require(::Module, ::Symbol) at .\loading.jl:922
│  [20] include(::String) at .\client.jl:439
│  [21] top-level scope at none:6
│  [22] eval(::Module, ::Any) at .\boot.jl:331
│  [23] exec_options(::Base.JLOptions) at .\client.jl:264
│  [24] _start() at .\client.jl:484
│ in expression starting at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\src\qps.jl:2
└ @ Requires C:\Users\Geoffroy Leconte\.julia\packages\Requires\9Jse8\src\require.jl:40
[ Info: Checking consistency of problem uncqp
ERROR: LoadError: LoadError: MethodError: no method matching QuadraticModel(::QPSData, ::Array{Float64,1})
Closest candidates are:
  QuadraticModel(::Any, ::Any, ::Any) at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\src\qpmodel.jl:17
  QuadraticModel(::AbstractNLPModel, ::AbstractArray{T,1} where T; kwargs...) at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\src\qpmodel.jl:83
Stacktrace:
 [1] uncqp_QPSData() at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\problems\uncqp.jl:35
 [2] top-level scope at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\test_consistency.jl:37
 [3] include(::String) at .\client.jl:439
 [4] top-level scope at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\runtests.jl:18
 [5] include(::String) at .\client.jl:439
 [6] top-level scope at none:6
in expression starting at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\test_consistency.jl:33
in expression starting at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\test\runtests.jl:18
ERROR: Package QuadraticModels errored during testing
```